cmd/tendermint/commands/run_node.go

```diff
@@ -6,20 +6,61 @@ import (
 
 	"github.com/spf13/cobra"
 
+	cmn "github.com/tendermint/tmlibs/common"
+
 	"github.com/tendermint/tendermint/node"
+	"github.com/tendermint/tendermint/proxy"
 	"github.com/tendermint/tendermint/types"
-	cmn "github.com/tendermint/tmlibs/common"
 )
 
-var runNodeCmd = &cobra.Command{
+// RunNodeCmd creates and starts a tendermint node.
+var RunNodeCmd = &cobra.Command{
```

Comment:
isn't this redundant if we have the function below?

Reply:
I wanted to keep it to not break the API, but I did it anyway so now it's gone :-)